app/text_query.py

```diff
@@ -13,6 +13,8 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+from __future__ import absolute_import
```

Comment:
Looks a bit weird that `__author__` is sandwiched with `import`s. Any reason to put those imports before `__author__`?

Reply:
Yeah, python-modernize expects it to be at the very top, which kinda makes sense (since it's something that affects the imports themselves), but it is a little bit silly that it thinks it matters where __author__ is. It's not just a matter of where it puts it by default, it'll also complain if you fix it manually, so if we want to use python-modernize to avoid regressions we have to tolerate this weirdness.

Reply:
I see. Is it still possible to put a blank line before `__author__` to make it look a bit less weird?

Or maybe we can just remove `__author__`. We don't add it for recent files...

Reply:
I added a blank line above `__author__` in PR #553.